js/app.js

```diff
@@ -0,0 +1,14 @@
+import { ToDoList } from "./ToDoList.js";
+
+const AddBtn = document.querySelector(".input-text button");
+const description_task = document.getElementById("task-input");
+
+document.addEventListener("DOMContentLoaded", () => {
+  AddBtn.addEventListener("click", () => {
+    if (description_task.value == "") {
+      alert("You should add a task!");
+    } else {
+      new ToDoList();
+    }
+  });
+});
```

Comment:
- why you're creating `new ToDoList()` with each new task!! All the app must have only one `ToDoList()`
please use something like below!
```
document.addEventListener("DOMContentLoaded", () => {
  const toDoList = new ToDoList();
  AddBtn.addEventListener("click", () => {
    if (description_task.value == "") {
      alert("You should add a task!");
    } else {
      toDoList.addTask(description_task.value);
    }
  });
});
```